libnavigation-core/src/test/java/com/mapbox/navigation/core/DeveloperMetadataAggregatorTest.kt

```diff
@@ -0,0 +1,110 @@
+package com.mapbox.navigation.core
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.testing.MainCoroutineRule
+import io.mockk.clearMocks
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableSharedFlow
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@OptIn(ExperimentalCoroutinesApi::class, ExperimentalPreviewMapboxNavigationAPI::class)
+class DeveloperMetadataAggregatorTest {
+
+    @get:Rule
+    val coroutineRule = MainCoroutineRule()
+    private val sessionIdFlow = MutableSharedFlow<String>(replay = 5)
```

Comment:
why is it 5?

Reply:
I was struggling with making flows work the way I want them to, it's not needed anymore.